inc/namespace.php

```diff
@@ -17,6 +17,11 @@
 function bootstrap() {
 	add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_notifications', 0 );
 	add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_publication_checklist', 0 );
+	add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_duplicate_posts' );
+	add_action( 'admin_menu', __NAMESPACE__ . '\\remove_duplicate_post_admin_page', 99 );
+	add_filter( 'manage_post_posts_columns', __NAMESPACE__ . '\\remove_duplicate_post_original_item_column', 11 );
+	add_filter( 'manage_page_posts_columns', __NAMESPACE__ . '\\remove_duplicate_post_original_item_column', 11 );
```

Comment:
No, but I could probably pull in `duplicate_post_enabled_post_types` after we filter that to get the enabled post types and loop through that instead of only affecting the default post types. I guess my thought here was if you were adding additional post types, you could choose whether to hide the column.